src/hotspot/share/opto/callGenerator.cpp

```diff
@@ -554,6 +554,53 @@ void LateInlineVirtualCallGenerator::do_late_inline() {
   CallGenerator::do_late_inline_helper();
 }
 
+static void delay_box_in_uncommon_trap(CallNode* call, Node* resproj) {
+  if (resproj != NULL && call->is_CallStaticJava() &&
```

Comment:
IMHO, we should use nullptr here because hotspot now is using c++14.

Reply:
Thank you for your review. I will change that.